components/TopicFunction.js

```diff
@@ -7,13 +7,21 @@ export function TopicFunction({ channels }) {
     const topicsDetails = getTopics(channels);
     let functions = '';
 
-    topicsDetails.forEach(t => {
-      functions += `def send${ t.name }(self, id):
+topicsDetails.forEach(t => {
+    let functionName = '';
+    if (t.name) {
+        functionName = t.name.charAt(0).toUpperCase() + t.name.slice(1);
+    } else {
+        functionName = t.topic.split('/').pop().replace(/[^a-zA-Z0-9]/g, '');
```

Comment:
it should work this way that in case of address `comment/unliked` the function name is `commentUnliked` and in case of `comment/unliked/sent` the function name is `commentUnlikedSent`